R/LSTdescriptives.R

```diff
@@ -302,31 +300,63 @@ LSTdescriptives <- function(jaspResults, dataset, options, state = NULL) {
   nearestPointIndex <- which.min(abs(valueSet - targetValue))
   return(nearestPointIndex)
 }
-.descExplanationCT <- function(jaspResults, options) {
+
+.descExplanationCT <- function(jaspResults, options, colors) {
   jaspResults[["descExplanationCT"]] <- createJaspContainer(gettext("Explanation"))
   jaspResults[["descExplanationCT"]]$position <- 1
   jaspResults[["descExplanationCT"]]$dependOn(options = c("LSdescCentralOrSpread",
                                                           "LSdescCT",
                                                           "LSdescExplanationC"))
   
+  pdPlot <- createJaspPlot(title = gettext("Theoretical example distribution"), width = 700, height = 400)
+  pdPlot$position <- 1
+  
   mean <- 0
   sd <- 1
   skew <- 1000
+  distLimits <- c(mean - 4 * sd, mean + 4 * sd)
   
-  pdPlot <- createJaspPlot(title = gettext("Theoretical example distribution"), width = 700, height = 400)
-  pdPlot$position <- 1
+  data <- data.frame(x = .scaledSkewedNormal(100000, xi = mean, omega = sd, alpha = skew))
+  data <- subset(data, data$x > distLimits[1] & data$x < distLimits[2]) # remove values outside limits
+  
+  pdPlotObject <- .plotCTexampleDistribution(data)
+  
+  allCTs <- options[["LSdescCT"]] == "LSdescMMM"
+  
+  if (options[["LSdescCT"]] == "LSdescMedian"| allCTs) {
+    pdPlotObject <- .visualExplanationMedian(pdPlotObject, data, options, colors)
+    text <- gettext("Text for Median:  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.")
```

Comment:
I assume that the Lorem ipsum is just a placeholder?

@boutinb do people already translate this module? Because if they do we might want to remove the `gettext` stuff for now so that this isn't shown to translators.

Reply:
Well, either way not wrong to remove it for now, so I'll do that.